docs/api-reference/tasks-api.md

```diff
@@ -25,77 +25,1448 @@ sidebar_label: Tasks
+* State: `complete`
+* Datasource: `wikipedia_api`
+* Time interval: between `2015-09-12` and `2015-09-13`
+* Max entries returned: `10` 
+* Task type: `query_worker`
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cURL-->
+```shell
+curl "{domain}/druid/indexer/v1/tasks/?state=complete&datasource=wikipedia_api&createdTimeInterval=2015-09-12T00%3A00%3A00Z%2F2015-09-13T23%3A59%3A59Z&max=10&type=query_worker"
+```
+<!--HTTP-->
+```HTTP
+GET /druid/indexer/v1/tasks/?state=complete&datasource=wikipedia_api&createdTimeInterval=2015-09-12T00%3A00%3A00Z%2F2015-09-13T23%3A59%3A59Z&max=10&type=query_worker HTTP/1.1
+Host: {domain}
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+#### Sample response
+
+<details>
+  <summary>Click to show sample response</summary>
+  
+  ```json
+  [
+    {
+        "id": "query-223549f8-b993-4483-b028-1b0d54713cad-worker0_0",
+        "groupId": "query-223549f8-b993-4483-b028-1b0d54713cad",
+        "type": "query_worker",
+        "createdTime": "2023-06-22T22:11:37.012Z",
+        "queueInsertionTime": "1970-01-01T00:00:00.000Z",
+        "statusCode": "SUCCESS",
+        "status": "SUCCESS",
+        "runnerStatusCode": "NONE",
+        "duration": 17897,
+        "location": {
+            "host": "localhost",
+            "port": 8101,
+            "tlsPort": -1
+        },
+        "dataSource": "wikipedia_api",
+        "errorMsg": null
+    },
+    {
+        "id": "query-fa82fa40-4c8c-4777-b832-cabbee5f519f-worker0_0",
+        "groupId": "query-fa82fa40-4c8c-4777-b832-cabbee5f519f",
+        "type": "query_worker",
+        "createdTime": "2023-06-20T22:51:21.302Z",
+        "queueInsertionTime": "1970-01-01T00:00:00.000Z",
+        "statusCode": "SUCCESS",
+        "status": "SUCCESS",
+        "runnerStatusCode": "NONE",
+        "duration": 16911,
+        "location": {
+            "host": "localhost",
+            "port": 8101,
+            "tlsPort": -1
+        },
+        "dataSource": "wikipedia_api",
+        "errorMsg": null
+    },
+    {
+        "id": "query-5419da7a-b270-492f-90e6-920ecfba766a-worker0_0",
+        "groupId": "query-5419da7a-b270-492f-90e6-920ecfba766a",
+        "type": "query_worker",
+        "createdTime": "2023-06-20T22:45:53.909Z",
+        "queueInsertionTime": "1970-01-01T00:00:00.000Z",
+        "statusCode": "SUCCESS",
+        "status": "SUCCESS",
+        "runnerStatusCode": "NONE",
+        "duration": 17030,
+        "location": {
+            "host": "localhost",
+            "port": 8101,
+            "tlsPort": -1
+        },
+        "dataSource": "wikipedia_api",
+        "errorMsg": null
+    }
+  ]
+  ```
+
+</details>
+
+### Get an array of complete tasks
+
+#### URL
+<code class="getAPI">GET</code> `/druid/indexer/v1/completeTasks`
+
+This endpoint retrieves an array of completed task objects in the current Druid cluster. This is functionally equivalent to `/druid/indexer/v1/tasks?state=complete`. It supports a set of optional query parameters to filter results. 
```

Comment:
```suggestion
This endpoint retrieves an array of completed tasks in the Druid cluster. This is functionally equivalent to `/druid/indexer/v1/tasks?state=complete`. It supports a set of optional query parameters to filter results. 
```

Reply:
It could be less wordy to refer to them as "tasks" instead of "task objects"

Reply:
Very handy to say what this endpoint is functionally equivalent to in the previous endpoint 👌